src/now.js

```diff
@@ -53,10 +53,16 @@ const main = async (argv_) => {
   // $FlowFixMe
   const { isTTY } = process.stdout
 
-  const update = await checkForUpdate(pkg, {
-    interval: ms('1d'),
-    distTag: pkg.version.includes('canary') ? 'canary' : 'latest'
-  })
+  let update = null
+
+  try {
+    update = await checkForUpdate(pkg, {
+      interval: ms('1d'),
+      distTag: pkg.version.includes('canary') ? 'canary' : 'latest'
+    })
+  } catch (err) {
```

Comment:
I don't have access to the `--debug` flag here, otherwise I'd show the full error in debug mode.